syntax/std_rel.go

```diff
@@ -9,7 +9,18 @@ func stdRel() rel.Attr {
 		rel.NewNativeFunctionAttr("union", func(v rel.Value) rel.Value {
 			s := v.(rel.Set)
 			sets := make([]rel.Set, 0, s.Count())
-			for e, ok := s.ArrayEnumerator(); ok && e.MoveNext(); {
+			var e rel.ValueEnumerator
+			switch u := v.(type) {
+			case rel.Array:
```

Comment:
Only sets should be accepted.

Reply:
treating array as set created complications as `Enumerator.Current()` would return a tuple `(@, @values)`

Reply:
That's as intended.

Reply:
right, some of the example uses `//rel.union(array)`, I'll change it